index.html

```diff
@@ -0,0 +1,105 @@
+                    <a class="nav-link" href="#contact-info">Contact Info</a>
+                </li>
+            </ul>
+        </div>     
+    </nav>
+    <div class="navbar-text ml-auto">
+        <span id="current-date"></span>
+      </div>
+    <div class="container mt-4">
+        <section id="book-list-c">
+            <h2 class="text-center mb-4"><i class="fas fa-book text-primary"></i> Books List</h2>
+            <table class="table table-striped">
+                <thead>
+                    <tr>
+                        <th>Title</th>
+                        <th>Author</th>
+                        <th>ISBN#</th>
+                        <th> 
+                        </th>
+                    </tr>
+                </thead>
+                <tbody id="book-list"></tbody>
+            </table>
+        </section>
+        <section id="add-book-form">
+            <h2 class="text-center mb-4"><i class="fas fa-plus text-primary"></i> Add Book Form</h2>
+            <form id="book-form">
+                <div class="form-group">
+                    <label for="title">Title</label>
+                    <input type="text" id="title" class="form-control">
+                </div>
+                <div class="form-group">
+                    <label for="author">Author</label>
+                    <input type="text" id="author" class="form-control">
+                </div>
+                <div class="form-group">
+                    <label for="isbn">ISBN#</label>
+                    <input type="text" id="isbn" class="form-control">
+                </div>
+                <input type="submit" value="Add Book" class="btn btn-primary btn-block">
+            </form>
+            
+        </section>
+          <!-- Contact Us section -->
+    <section id="contact-info" class="container mt-4">
+      <h2 class="display-4 text-center mb-4">Contact Us</h2>
+      <form>
+        <div class="form-group">
+          <label for="name">Name</label>
+          <input type="text" id="name" class="form-control">
+        </div>
+        <div class="form-group">
+          <label for="email">Email</label>
+          <input type="email" id="email" class="form-control">
+        </div>
+        <div class="form-group">
+          <label for="message">Message</label>
+          <textarea id="message" class="form-control"></textarea>
+        </div>
+        <input type="submit" value="Send" class="btn btn-primary btn-block">
+      </form>
+    </section>
+    </div>
+    <script src="index.js" type="module"></script>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.3.0/build/global/luxon.min.js"></script>
+<script>
+  // Get the current date and format it using Luxon
+  const currentDate = luxon.DateTime.local().toFormat('EEEE, LLLL d, y');
+
+  // Update the current date element with the formatted date
+  const currentDateElement = document.getElementById('current-date');
+  currentDateElement.innerText = currentDate;
+</script>
```

Comment:
- [ ] The purpose of this project is to start developing the SOLID mentality and modularize your project so it become more organized. The date is correctly displayed on the UI, but kindly import the module from Luxon in your `index.js` file, e.g:`import { DateTime } from "luxon";`. Please, download it using `npm` and use it as a module as the project required.